components/work_templates/components/menu.tsx

```diff
@@ -70,21 +69,23 @@ const Menu = (props: MenuProps) => {
     const worktemplates = useSelector((state: GlobalState) => state.entities.worktemplates.templatesInCategory);
 
     useEffect(() => {
-        // DEV - REMOVE ME WHEN SERVER RETURNS DATA
-        // allows us to load the dev work templates from the store
-        dispatch(injectDevData(workTemplateDevData));
+        if (categories?.length) {
+            return;
+        }
+        dispatch(getWorkTemplateCategories());
     }, []);
 
     useEffect(() => {
-        if (categories?.length === 0 || selectedCategory !== '') {
+        if (categories?.length === 0) {
```

Comment:
I'd replace this for `(!categories?.length)` that would cover you against categories being undefined
<img width="342" alt="Captura de Pantalla 2022-12-11 a las 21 19 44" src="https://user-images.githubusercontent.com/10082627/206926763-8c51fcaf-69e8-4aa8-aeaf-b18f5cce2d1e.png">